### PR TITLE
Add karmadactl interpret examples

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind feature
/kind documentation

**What this PR does / why we need it**:

Add `karmadactl interpret` command usage examples.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: examples/karmadactlinterpret/resourceinterpretercustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+apiVersion: config.karmada.io/v1alpha1
+kind: ResourceInterpreterCustomization
+metadata:
+  name: declarative-configuration-example
+spec:
+  target:
+    apiVersion: apps/v1
+    kind: Deployment
+  customizations:
+    retention:
```
Comment (ikaven1024) [inline examples/karmadactlinterpret/resourceinterpretercustomization.yaml:10]:
```suggestion
```

File: examples/karmadactlinterpret/resourceinterpretercustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          desiredObj.status.replicas = replicas
+          return desiredObj
+        end
+    statusReflection:
+      luaScript: >
+        function ReflectStatus (observedObj)
+          if observedObj.status == nil then
+            return nil
+          end
+        return observedObj.status
```
Comment (ikaven1024) [inline examples/karmadactlinterpret/resourceinterpretercustomization.yaml:54]:
```suggestion
          return observedObj.status
```

File: examples/karmadactlinterpret/resourceinterpretercustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        function AggregateStatus(desiredObj, statusItems)
+          if statusItems == nil then
+            return desiredObj
+          end
+          if desiredObj.status == nil then
+            desiredObj.status = {}
+          end
+          replicas = 0
+          for i = 1, #statusItems do
+            if statusItems[i].status ~= nil and statusItems[i].status.replicas ~= nil  then
```
Comment (ikaven1024) [inline examples/karmadactlinterpret/resourceinterpretercustomization.yaml:41]:
```suggestion
            if statusItems[i].status ~= nil and statusItems[i].status.replicas ~= nil then
```

**Review by ikaven1024 — COMMENTED** [review]
Others LGTM

File: examples/karmadactlinterpret/resourceinterpretercustomization.yaml (hunk 1 above)
Comment (XiShanYongYe-Chang) [reply examples/karmadactlinterpret/resourceinterpretercustomization.yaml]: Thanks~.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**